components/collapsible-panel/test/collapsible-panel.visual-diff.js

```diff
@@ -20,8 +19,8 @@ describe('d2l-collapsible-panel', () => {
 
 	after(async() => await browser.close());
 
-	function focusElement(selector) {
-		return page.$eval(selector, (elem) => forceFocusVisible(elem));
+	async function focusElement(selector) {
+		return focusWithKeyboard(page, `${selector} > d2l-collapsible-panel`);
```

Comment:
Does this function need to be `async`?

Reply:
Hmm, maybe not. I had it `await focusWithKeyboard`-ing at one point while I was debugging, which would have required the `async`. But I think as it is now, it can be removed.

Reply:
Yep returning an async function will pass along the promise, `async` is only necessary when directly using `await`.